app/services/mobilize.py

```diff
@@ -0,0 +1,81 @@
+#  MIT License
+#
+#  Copyright (c) 2020 Daniel C. Brotsky
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to deal
+#  in the Software without restriction, including without limitation the rights
+#  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#  copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in all
+#  copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+#  SOFTWARE.
+
+import asyncio
+import pickle
+import pandas as pd
+
+from typing import Dict
+from fastapi import APIRouter
+from fastapi import FastAPI, File, UploadFile, Request
+from fastapi.responses import HTMLResponse
+from fastapi.staticfiles import StaticFiles
+from fastapi.templating import Jinja2Templates
+from starlette.responses import JSONResponse
+
+from ..utils import prinl, log_error, MapContext, env, Timestamp
+from ..db import redis, ItemListStore as Store
+from ..workers import process_all_item_lists
+
+mobilize = APIRouter()
+mobilize.mount("/static", StaticFiles(directory="static"), name="static")
```

Comment:
You don't actually need this yet, because you're not using any static files.  But see the comments below: your current upload form is really a static file, and you will also eventually want statics for branding purposes.

You probably want to put the static mount in the `main.py` file since it establishes a root directory.  Also I recommend a mount point of "/docs" rather than "/static".